apm-protocol/apm-network/pom.xml

```diff
@@ -75,7 +73,6 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-tcnative-boringssl-static</artifactId>
-            <version>${netty-tcnative-boringssl-static.version}</version>
```

Comment:
This is to intentionally override the one that brought in by the netty-bom, please make sure the version of native-boringssl is still this one after removing the version here